client.go

```diff
@@ -107,16 +100,27 @@ func newRPC2Client(conn net.Conn) (*OvsdbClient, error) {
 
 	// Process Async Notifications
 	dbs, err := ovs.ListDbs()
-	if err == nil {
-		for _, db := range dbs {
-			schema, err := ovs.GetSchema(db)
-			if err == nil {
-				ovs.Schema[db] = *schema
-			} else {
-				return nil, err
-			}
+	if err != nil {
+		c.Close()
+		return nil, err
+	}
+
+	for _, db := range dbs {
+		schema, err := ovs.GetSchema(db)
+		if err == nil {
+			ovs.Schema[db] = *schema
+		} else {
+			c.Close()
```

Comment:
@vtolstov how do you like the error from c.Close() be handled? We are already on the error path and if we get errors from c.Close(), then we will have two errors. Obviously, we would want to return the initial error and not the error from c.Close(). Any thoughts? Thanks.

Reply:
@vtolstov how do you want to proceed?

Reply:
I think that we need some logger interface that able to log message in case of Close() error and return major error that.
So for this pr we can merge as is now, and for next stuff create logger interface with some DefaultLogger (that can be overwritten). so all users of package can see problems and do whatever they wants